pkg/operator/resources/namespaced/uploadproxy.go

```diff
@@ -131,6 +132,13 @@ func createUploadProxyDeployment(image, verbosity, pullPolicy, priorityClassName
 			ReadOnly:  true,
 		},
 	}
+	container.Resources = corev1.ResourceRequirements{
+		Requests: corev1.ResourceList{
+			corev1.ResourceCPU:    resource.MustParse("10m"),
+			corev1.ResourceMemory: resource.MustParse("150Mi"),
```

Comment:
How are these numbers calculated?

Reply:
Picked the same numbers are the request sizes kubevirt is using. Note these are requests (aka minimum required), in reality the used amount will be larger.